package.json

```diff
@@ -6,14 +6,11 @@
     "@testing-library/jest-dom": "^5.16.2",
     "@testing-library/react": "^12.1.3",
     "@testing-library/user-event": "^13.5.0",
-    "ag-grid-community": "^27.1.0",
-    "ag-grid-react": "^27.1.0",
     "classnames": "^2.3.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-icons": "^4.3.1",
     "react-scripts": "5.0.0",
-    "styled-components": "^5.3.5",
```

Comment:
Still do not see styled-components used. You need to use it per task assignment, at least at minumum possible level to confirm you've got aknowledged with it.